### ISSUE TITLE
GBA Video - Shrek doesn’t load

### ISSUE THREAD

**Issue by tjanas** [MiSTer-devel/GBA_MiSTer#95]
As confirmed by a few users on MiSTer FPGA discord, trying to load this rom results in a blank white screen.

ROM verified to match [no-intro](https://datomatic.no-intro.org/index.php?page=show_record&s=23&n=2872).

**Comment by RobertPeip** [comment]
Yes, all 64MByte ROMs (only Videos as far as i know) are not supported currently.
They most likely need a special mapper.

**Comment by RobertPeip** [comment]
more info here: https://mgba.io/2015/10/20/dumping-the-undumped/

**Comment by RobertPeip** [comment]
mGBA implementation:
https://github.com/mgba-emu/mgba/commit/38e3dbc0fcec4356fff4212dee2c905d494f6c69

games:
Shark Tale (USA) (Rev 5).gba
Shark Tale (USA) (Rev 6).gba
Shrek (USA) (Rev 6).gba
Shrek + Shark Tale (USA) (Rev 5).gba
Shrek 2 (USA) (Rev 6).gba

**Comment by 0xc0ffea** [comment]
I get this is dumb .. are we really going to enjoy watch Shrek on GBA potato vision video .. really really .. yeah. probably.
